conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: numpy
+  version: "" # Set at build time.
+
+source:
+  git_url: ..
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openblas    # [linux]
+    - python
+    - cython
+    - gcc         # [unix]
+
+  run:
+    - openblas    # [linux]
```

Comment:
Turns out a version exists for OS X now, as well. Still don't see a copy provided for Windows. So will update this to `# [unix]`